app/models/user.rb

```diff
@@ -193,4 +224,15 @@ def generate_authentication_token
       break token unless User.find_by(authentication_token: token)
     end
   end
+
+  # Set up Auth0 management client for actions like adding users.
+  # See: https://github.com/auth0/ruby-auth0/blob/master/README.md#management-api-v2
+  private_class_method def self.auth0_management_client
+    @auth0_management_client ||= Auth0Client.new(
+      client_id: ENV["AUTH0_MANAGEMENT_CLIENT_ID"],
+      client_secret: ENV["AUTH0_MANAGEMENT_CLIENT_SECRET"],
+      domain: ENV["AUTH0_DOMAIN"],
```

Comment:
It would worth verifying if using custom domains here (like `login.idseq.net`) is fine, but I guess you may need to use tenant url here (ex: `czi-idseq-dev.auth0.com`). So you may need to set a `ENV["AUTH0_MANAGEMENT_DOMAIN"]`.

Reply:
will update 👍

Reply:
added `ENV["AUTH0_MANAGEMENT_DOMAIN"]` 👍